packages/fiori/src/themes/ShellBar.css

```diff
@@ -219,6 +218,10 @@ slot[name="profile"] {
 	width: 2.5rem;
 }
 
+.ui5-shellbar-button[focused]::part(button):after {
```

Comment:
Why do we need that specific selector? I think that the following code is enough:
```css
.ui5-shellbar-button {
    --_ui5_button_focused_border: var(--_ui5_shellbar_button_focused_border);
}
```

Reply:
Well, because I am trying to overwrite the default styles in `Button.css`

```css
:host([focused]) .ui5-button-root::after {}
```